src/app/_blocks/AuthorContentGrid/index.tsx

```diff
@@ -18,9 +18,7 @@ export default function AuthorContentGrid({content}) {
       <div className={styles.contentGrid}>
         {Object.keys(content).map(key =>
           content[key].map((contentPiece, i) => (
-            <div key={i} className={styles.contentCard}>
-              <ContentCard contentType={key} content={contentPiece} />
-            </div>
+            <ContentCard contentType={key} content={contentPiece} rounded={true}/>
```

Comment:
When passing boolean = true props to components, you can use shorthand, just 
```
<ContentCard {...other stuff} rounded />
```

Reply:
noted!